test_regress/t/t_output_concatenation_groups_splitting.dat

```diff
@@ -0,0 +1,338 @@
+           0	dummy_cpp_file_0000.seq_00_2x0_0.score_0
```

Comment:
Please have the generator put a comment up front e.g. "# Generated by nodist/gen_scores", and have parser ignore comment lines starting with #.

Should Verilator be able to generate a file like this itself, so it can be round-tripped?

Reply:
I decided that this kind of a test would be better off as a simple self-test. It is probably not worth it to add a special test input generator, a special format, a special debugging flag and special define logic just for this one feature.